plugins/org.python.pydev/tests_completions/org/python/pydev/ast/codecompletion/revisited/FindActualDefinitionTest.java

```diff
@@ -141,4 +150,26 @@ public void testFindActualDefinitionImported() throws Exception {
                 completionCache);
         assertEquals(1, selected.size());
     }
+
+    public void testOccurrencesForDefinition() throws Exception {
+        int usedGrammar = GRAMMAR_TO_USE_FOR_PARSING;
+        GRAMMAR_TO_USE_FOR_PARSING = PythonNature.LATEST_GRAMMAR_PY3_VERSION;
```

Comment:
This should be in a try..finally (i.e.: even if the test fails it should go back to the original value).